### ISSUE TITLE
USWDS-Site: Publish step indicator accessibility checklist page

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds-site#2668]
## Summary
Publish the `usa-step-indicator` accessibility test checklist items found in the [Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing) (Google Doc :lock:). 

The process for creating a new accessibility test page can be found in the [Creating accessibility checklists - process doc](https://docs.google.com/document/d/14WTCsdzgwpoz_Miztd66RmEGfwwSEUFCVN1DkSQITTE/edit#heading=h.fa3sv1ray42g) (Google Doc :lock:).

**Comment by amyleadem** [comment]
Marking this as hold pending the completion of uswds/uswds#6321

**Comment by amyleadem** [comment]
@RachelCorsino  Moving this to "on deck" status now that #3073 is complete. This is ready to build!

**Comment by amyleadem** [comment]
@dct2023 Reassigning this to myself to finish up since @RachelCorsino is out. Should be just a few minutes of work left.
